src/settings-ui/Settings.UI/ViewModels/ZoomItViewModel.cs

```diff
@@ -0,0 +1,761 @@
+
+            GeneralSettingsConfig = settingsRepository.SettingsConfig;
+
+            var zoomItSettings = global::PowerToys.ZoomItSettingsInterop.ZoomItSettings.LoadSettingsJson();
+            _zoomItSettings = JsonSerializer.Deserialize<ZoomItSettings>(zoomItSettings, _serializerOptions);
+
+            InitializeEnabledValue();
+
+            // set the callback functions value to handle outgoing IPC message for the enabled value.
+            SendConfigMSG = ipcMSGCallBackFunc;
+
+            // set the callback for when we need the user to pick a file.
+            PickFileDialog = pickFileDialog;
+
+            // set the callback for when we need the user to pick a font.
+            PickFontDialog = pickFontDialog;
+
+            _typeFont = TypeFont;
+
+            SelectDemoTypeFileCommand = new ButtonClickCommand(SelectDemoTypeFileAction);
+            SelectBreakSoundFileCommand = new ButtonClickCommand(SelectBreakSoundFileAction);
+            SelectBreakBackgroundFileCommand = new ButtonClickCommand(SelectBreakBackgroundFileAction);
+            SelectTypeFontCommand = new ButtonClickCommand(SelectTypeFontAction);
+
+            LoadMicrophoneList();
+        }
+
+        private void InitializeEnabledValue()
+        {
+            _enabledGpoRuleConfiguration = GPOWrapper.GetConfiguredZoomItEnabledValue();
+            if (_enabledGpoRuleConfiguration == GpoRuleConfigured.Disabled || _enabledGpoRuleConfiguration == GpoRuleConfigured.Enabled)
+            {
+                // Get the enabled state from GPO.
+                _enabledStateIsGPOConfigured = true;
+                _isEnabled = _enabledGpoRuleConfiguration == GpoRuleConfigured.Enabled;
+            }
+            else
+            {
+                _isEnabled = GeneralSettingsConfig.Enabled.ZoomIt;
+            }
+        }
+
+        private void SendCustomAction(string actionName)
+        {
+            SendConfigMSG("{\"action\":{\"ZoomIt\":{\"action_name\":\"" + actionName + "\", \"value\":\"\"}}}");
+        }
+
+        public bool IsEnabled
+        {
+            get => _isEnabled;
+
+            set
+            {
+                if (_enabledStateIsGPOConfigured)
+                {
+                    // If it's GPO configured, shouldn't be able to change this state.
+                    return;
+                }
+
+                if (value != _isEnabled)
+                {
+                    _isEnabled = value;
+
+                    // Set the status in the general settings configuration
+                    GeneralSettingsConfig.Enabled.ZoomIt = value;
+                    OutGoingGeneralSettings snd = new OutGoingGeneralSettings(GeneralSettingsConfig);
+
+                    SendConfigMSG(snd.ToString());
+                    OnPropertyChanged(nameof(IsEnabled));
+                }
+            }
+        }
+
+        public bool IsEnabledGpoConfigured
+        {
+            get => _enabledStateIsGPOConfigured;
```

Comment:
```suggestion
            get => _enabledStateIsGPOConfigured && _isEnabled;
```

Reply:
I don't think I agree with the change. We want this to show whether it's enabled or not.

Reply:
You are right. It is the enabled policy and not a utility behavior policy. I had behavior policies in mind when writing the review.